app/views/admin/communities/new_layout.haml

```diff
@@ -17,8 +17,7 @@
   .row
     .col-12
       %h2= t("admin.communities.new_layout.new_layout")
-      %p= t("admin.communities.new_layout.description_roadmap")
-      %p= t("admin.communities.new_layout.description_process", support_link: support_link).html_safe
```

Comment:
`support_link` is not used anymore, so it should be removed also from the line 7.